src/StructTypes.jl

```diff
@@ -111,6 +127,7 @@ Struct() = UnorderedStruct()
 StructType(u::Union) = Struct()
 StructType(::Type{Any}) = Struct()
 StructType(::Type{<:NamedTuple}) = Struct()
+StructType(::Type{DataType}) = Struct()
```

Comment:
This definition doesn't really work because, albeit confusingly, StructTypes.jl has its own definition of `DataType`, which `Struct`, `Mutable`, etc. subtype. I'm not super clear on why we would want this definition anyway? What's the use-case?

Reply:
You're right, this is not correct

Reply:
Removed